scripts/install_xcpng.py

```diff
@@ -230,11 +240,15 @@ def main():
         host2 = pool2.master
         host2.inventory = host2._get_xensource_inventory()
         check_mac_address(host2, mac_address)
-        logging.info('New host is started and enabled')
-        if host2.xcp_version != version.parse(xcp_version):
+        logging.info(f'Target host is started and enabled in version:{host2.xcp_version}')
+        if args.action == 'restore' and not host2.xcp_version < version.parse(xcp_version):
```

Comment:
```suggestion
        if args.action == 'restore' and host2.xcp_version >= version.parse(xcp_version):
```
Simpler & clearer :)